SimG4CMS/HcalTestBeam/plugins/HcalTB04Analysis.cc

```diff
@@ -659,7 +661,9 @@ void HcalTB04Analysis::fillBuffer(const EndOfEvent* evt) {
     hite[j] = &ehits[j];
   }
   sort(hite.begin(), hite.end(), CaloHitIdMore());
-  nhit = 0;
+#ifdef EDM_ML_DEBUG
+  int nhit = 0;
```

Comment:
@aandvalenzuela , this does not look rigth. I think it should be `nhit=0;`  otherwise in DBG IBs it will generate error like
```
/build/muz/dl/CMSSW_13_2_DBG_X_2023-05-04-2300/src/SimG4CMS/HcalTestBeam/plugins/HcalTB04Analysis.cc: In member function 'void HcalTB04Analysis::fillBuffer(const EndOfEvent*)':
/build/muz/dl/CMSSW_13_2_DBG_X_2023-05-04-2300/src/SimG4CMS/HcalTestBeam/plugins/HcalTB04Analysis.cc:665:7: error: redeclaration of 'int nhit'
  665 |   int nhit = 0;
      |       ^~~~
/build/muz/dl/CMSSW_13_2_DBG_X_2023-05-04-2300/src/SimG4CMS/HcalTestBeam/plugins/HcalTB04Analysis.cc:547:7: note: 'int nhit' previously declared here
  547 |   int nhit = 0;
      |       ^~~~
/build/muz/dl/CMSSW_13_2_DBG_X_2023-05-04-2300/src/SimG4CMS/HcalTestBeam/plugins/HcalTB04Analysis.cc: In member function 'void HcalTB04Analysis::fillEvent(PHcalTB04Info&)':
/build/muz/dl/CMSSW_13_2_DBG_X_2023-05-04-2300/src/SimG4CMS/HcalTestBeam/plugins/HcalTB04Analysis.cc:1002:7: error: redeclaration of 'int i'
 1002 |   int i = nhit;
      |       ^
/build/muz/dl/CMSSW_13_2_DBG_X_2023-05-04-2300/src/SimG4CMS/HcalTestBeam/plugins/HcalTB04Analysis.cc:983:7: note: 'int i' previously declared here
  983 |   int i = 0;
      |       ^
```